webplugin/js/app/kommunicate.js

```diff
@@ -124,8 +124,10 @@ $applozic.extend(true, Kommunicate, {
     updateConversationMetadata: function (conversationMetadata) {
         if (conversationMetadata) {
             if (
-                kommunicateCommons.isObject(conversationMetadata) &&
-                kommunicateCommons.isObject(conversationMetadata.metadata) &&
+                Object.prototype.toString.call(conversationMetadata) ==
```

Comment:
do we need this? or we just need to check if conversationMedta is a JSON?

Reply:
`conversationMetadata` has to be an object and `conversationMetadata.metadata` can be an object or in JSON format.
This is to prevent the pass an array or string for `conversationMetadata.metadata` property.

Reply:
this will work in IE?

also can you attach same sample codes that customers will use

Reply:
Sample code after patching: https://jsfiddle.net/m3kwbq49/
For IE: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString#browser_compatibility

Reply:
jsfiddle is showing error, i guess that evn doesn't have this fix deployed on it?